src/utils/parseError.ts

```diff
@@ -150,6 +158,14 @@ export const parseError = async (
         e.stack
       )
     }
+
+    if (e.Code === LifiErrorCode.ValidationError) {
```

Comment:
Not sure if it makes a difference but it should probably be `e.code`

Reply:
that does make a difference. I just copied the code from the transaction unprepared check. There it is `e.Code`. So we are getting parsed messages for this error now too :)